packages/nux/src/components/inline-tip/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { Notice } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+export function InlineTip( { isVisible, className, onRemove, children } ) {
+	if ( ! isVisible ) {
+		return null;
+	}
+
+	return (
+		<Notice className={ className } onRemove={ onRemove }>
+			{ children }
+		</Notice>
+	);
+}
+
+export default compose(
+	withSelect( ( select, { tipId } ) => ( {
+		isVisible: select( 'core/nux' ).isTipVisible( tipId ),
+	} ) ),
+	withDispatch( ( dispatch, { tipId } ) => {
+		const { disableTips, dismissTip } = dispatch( 'core/nux' );
+
+		return {
+			onRemove() {
+				// Disable reason: We don't yet have a <Confirm> component. One day!
+				// eslint-disable-next-line no-alert
+				if ( window.confirm( __( 'Would you like to disable tips like these in the future? ' ) ) ) {
```

Comment:
I'd like to replace this with a `Modal` but am running into an issue where clicking on the modal dismisses the inserter popover. Possibly related to #14851.

I'm on the fence as to whether it's a blocker for this PR. On the one hand, it's a poor UX, but on the other, we use `confirm()` somewhat extensively throughout in the app (see https://github.com/WordPress/gutenberg/issues/16583).

Reply:
Confirmed that #14851 fixes this issue, so I'll try to get that merged.

Reply:
👌 I moved from ra `confirm()` to a `Modal` in b1270ca23f8231e58a905244d1fbca4d7a026caa.